tools/data_scripts/checksum_fn.sh

```diff
@@ -7,7 +7,7 @@ calculate_sha1sum() {
     echo $'\nCalculating checksum for' ${CHECKSUM_FOLDER}
     echo "Saving to ${SAVE_TO_PATH}"
     find $CHECKSUM_FOLDER -type f ! -name '*checksum*' ! -name '*MD*' \
-       -not -path '*/\.*' -printf "%P\n" | sort -z | sha1sum | tr -d '-' \
```

Comment:
I added `-printf "%P/n"` so that the checksum was based on the relative folder structure and not absolute. For example, the checksum for the datasets folder should be same irrespective of where the folder lives (in craftassist/agent or locobot/agent or anywhere else). Does this new solution achieve that? What is the issue exactly that you were running into?